### PR TITLE
bluez: 5.43 -> 5.45

### PR DESCRIPTION
**Pull request by armijnhemel**
###### Motivation for this change

new version, also gatttool is no longer built by default

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [X] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by asymmetric** [comment]
Should we completely remove the line then, or is it best practice to comment it out?

**Comment by armijnhemel** [comment]
I am not the maintainer and I am not sure whether or not gatttool is actually used by someone, so I just left it in there for now. If no one is using it, then probably best to just remove the line completely.

**Comment by asymmetric** [comment]
Or at least add a comment in the code explaining the situation.

**Comment by armijnhemel** [comment]
Added a comment about the commented line.

**Review by asymmetric — COMMENTED** [review]
LGTM, but I'll wait to see if the other people mentioned in the PR know what the best thing to do is here.

**Comment by armijnhemel** [comment]
Any news?

**Review by 0xABAB — APPROVED** [review]

**Comment by fpletz** [comment]
There are links in bin that point to wrong paths (../test):

```
total 941
-r-xr-xr-x  2 root root 195392 Jan  1  1970 bccmd
-r-xr-xr-x  2 root root  43400 Jan  1  1970 bluemoon
-r-xr-xr-x  2 root root 166744 Jan  1  1970 bluetoothctl
lrwxrwxrwx 11 root root     31 Jan  1  1970 bluetoothd -> ../libexec/bluetooth/bluetoothd
lrwxrwxrwx 11 root root     20 Jan  1  1970 bluez-list-devices -> ../test/list-devices
lrwxrwxrwx 11 root root     25 Jan  1  1970 bluez-monitor-bluetooth -> ../test/monitor-bluetooth
lrwxrwxrwx 11 root root     20 Jan  1  1970 bluez-simple-agent -> ../test/simple-agent
lrwxrwxrwx 11 root root     20 Jan  1  1970 bluez-test-adapter -> ../test/test-adapter
lrwxrwxrwx 11 root root     19 Jan  1  1970 bluez-test-device -> ../test/test-device
lrwxrwxrwx 11 root root     24 Jan  1  1970 bluez-test-thermometer -> ../test/test-thermometer
-r-xr-xr-x  2 root root  33944 Jan  1  1970 btattach
-r-xr-xr-x  2 root root 666392 Jan  1  1970 btmon
-r-xr-xr-x  2 root root  17824 Jan  1  1970 hex2hcd
-r-xr-xr-x  2 root root 100048 Jan  1  1970 l2ping
-r-xr-xr-x  2 root root 118416 Jan  1  1970 l2test
-r-xr-xr-x  2 root root 109768 Jan  1  1970 mpris-proxy
lrwxrwxrwx 11 root root     26 Jan  1  1970 obexd -> ../libexec/bluetooth/obexd
-r-xr-xr-x  2 root root 153888 Jan  1  1970 rctest
```

**Comment by fpletz** [comment]
Obsolete due to dadb16a57f6b13393d5f34d0e009ec0c2e50079b.

**Pull request closed without merging**
